.github/workflows/ci.yaml

```diff
@@ -5,45 +5,47 @@ on:
       - develop
       - release/**
       - hotfix/**
-      - feat/dev-583-remove-set-env-in-github-actions
+      - feat/DEV-2479-upgrade-py38
```

Comment:
Is this temporary?

Reply:
yes, i can remove all the feature branches here once all the PR changes are done.